src/containers/tools/note/index.tsx

```
@@ -85,7 +91,8 @@ const ToolsNoteScreen = () => {
             <Typography.Text>
               {countingType === 'words' ? length : (editor?.getText().length ?? 0)} {countingType}
             </Typography.Text>
-          </Button>
+          </Button>,
+          <Button type='default' shape='circle' icon={<CopyOutlined />} onClick={onCopy} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)